core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -68,12 +68,24 @@
 public final class Configuration {
   private static final Logger LOG = LoggerFactory.getLogger(Configuration.class);
 
+  /** The source of a configuration property. */
+  public enum Source {
+    UNKNOWN,
```

Comment:
describe the priority order? Or use alphabetical order

Reply:
done